app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+  Strong_param_user = %i(name email password password_confirmation).freeze
```

Comment:
tên biến nên gọn thôi, ko cần trích dẫn user nữa đâu e, vì đang trong model user rồi
```suggestion
  ATTR_PARAMS = %i(name email password password_confirmation).freeze
```